web_src/js/components/DashboardRepoList.vue

```diff
@@ -255,6 +256,40 @@ const sfc = {
     $(el).find('.dropdown').dropdown();
     nextTick(() => {
       this.$refs.search.focus();
+      this.$refs.search.addEventListener('keydown', (e) => {
```

Comment:
Can the event listener be added by Vue way, the `@` ?

Reply:
Yes it should, vue style listener also will take care of `removeEventListener` automatically which this doesn't.